osbs/build/spec.py

```diff
@@ -258,6 +260,7 @@ def set_params(self, sources_command=None, architecture=None, vendor=None,
         self.koji_target.value = koji_target
         self.kojiroot.value = kojiroot
         self.kojihub.value = kojihub
+        self.koji_certs_secret = koji_certs_secret
```

Comment:
I just understood what you told me earlier - this comes from another PR. I've added comments there about the two instancesides of missing `.value`, and you'll be able to re-base once they're fixed.

In the meantime you could add the fixes to this pull request as a separate commit that will be easy to skip on re-base, just for the purpose of getting the Travis testing to pass.